plugin/storage/kafka/options.go

```diff
@@ -36,10 +36,14 @@ const (
 	suffixBrokers  = ".brokers"
 	suffixTopic    = ".topic"
 	suffixEncoding = ".encoding"
+	suffixRequiredAcks = ".requiredacks"
```

Comment:
nit: Personally would prefer `.required.acks` - similar to the `request.required.acks` mentioned in the godocs.